STROOP/Forms/StroopMainForm.cs

```diff
@@ -100,6 +100,16 @@ private void InitTabs()
             }
         }
 
+        private string GetDisplayNameForProcess(Process process)
+        {
+            if (string.IsNullOrWhiteSpace(process.MainWindowTitle))
+            {
+                return $"{process.ProcessName} ({process.Id})";
+            }
+            
+            return process.MainWindowTitle;
```

Comment:
I think I'd personally prefer to write this is a ternary expression, but that's not exactly an issue.